tools/topology/topology2/include/pipelines/cavs/passthrough-capture-be.conf

```diff
@@ -0,0 +1,68 @@
+#
+# BE DAI passthrough pipeline for capture
+#
+# All attributes defined herein are namespaced by alsatplg to
+# "Object.Pipeline.passthrough-be-capture.N.attribute_name"
+#
+# Usage: mixin-playback pipeline object can be instantiated as:
```

Comment:
Oops, copy'n'paste error in the doc, will fix in later version.

Reply:
Fix incrementally in another PR.